packages/notifi-react-card/lib/components/subscription/defaults.css

```diff
@@ -53,7 +54,16 @@
 
 .NotifiIntercomCard__container {
   background-color: var(--notifi-intercom-card-background);
-  box-shadow: 0px 4px 30px 0px #0000000A;
+  box-shadow: 0px 4px 30px 30px #0000000A;
+  width: 364px;
+}
+
+.NotifiIntercomCard__chatWindowContainer {
```

Comment:
We should add some padding between the message box and messages themselves. Could you add some padding? Not 100% sure if it would be here though haha